doc/contributing/documentation_guide.md

```diff
@@ -127,7 +127,14 @@ a             #=> [2, 3, 1]
 
 ### Headings
 
-Organize a long discussion with [headings](rdoc-ref:RDoc::MarkupReference@Headings).
+Organize a long discussion for a class or module with [headings](rdoc-ref:RDoc::MarkupReference@Headings).
+
+Do not use formal headings in the documentation for a method or constant.
+
+In the rare case where heading-like structures are needed
+within the documentation for a method or constant, use
+[bold text](rdoc-ref:RDoc::MarkupReference@Bold)
+as pseudo-headings.
```

Comment:
I'm curious about the rationale of this sentence.
What is the criteria between headings and heading-like structures?
What's wrong with lower level headings, like `====` or more?